src/main/java/org/jenkinsci/plugins/github_branch_source/GitHubBuildStatusNotificationBundle.java

```diff
@@ -0,0 +1,62 @@
+package org.jenkinsci.plugins.github_branch_source;
+
+import org.kohsuke.github.GHCommitState;
+
+public class GitHubBuildStatusNotificationBundle {
+
+    private final String context;
```

Comment:
Use `@CheckForNull` and `@NonNull` to document nullability (and use the ones from findbugs... yes they are currently marked as deprecated because the authors do not understand the Oracle JRE redistribution policy and didn't bother pushing their JSR to publish something)